### PR TITLE
54544 - Pending appointments content changes

### PR DESCRIPTION
**Pull request by vbahinwillit**
## Summary
This PR fixes column layout and content.

## Related issue(s)
- department-of-veterans-affairs/va.gov-team#54544

## Testing done
Unit and e2e testing

## Screenshots
localhost_3001_health-care_schedule-view-va-appointments_appointments_pending (1)

## What areas of the site does it impact?
VAOS

## Acceptance criteria

- [ ]  I fixed|updated|added unit tests and integration tests for each feature (if applicable).
- [ ]  No error nor warning in the console.
- [ ]  Events are being sent to the appropriate logging solution
- [ ]  Documentation has been updated (link to documentation)
- [ ]  No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs
- [ ]  Feature/bug has a monitor built into Datadog or Grafana (if applicable)
- [ ]  If app impacted requires authentication, did you login to a local build and verify all authenticated routes work as expected
- [ ]  I added a screenshot of the developed feature
- [ ]  [Accessibility foundational testing](https://depo-platform-documentation.scrollhelp.site/developer-docs/wcag-2-1-success-criteria-and-foundational-testing) has been performed

## Requested Feedback

(OPTIONAL)_What should the reviewers know in addition to the above. Is there anything specific you wish the reviewer to assist with. Do you have any concerns with this PR, why?_

### CODE REVIEW

File: src/applications/vaos/appointment-list/components/AppointmentsPageV2/RequestAppointmentLayout.jsx [hunk 1]
```diff
@@ -68,9 +71,6 @@ export default function RequestAppointmentLayout({ appointment }) {
                     {`${modality}`}
```
Comment (jenniemc) [inline src/applications/vaos/appointment-list/components/AppointmentsPageV2/RequestAppointmentLayout.jsx:71]: modality contains the returned value from the function `selectModalityText(appointment)` Currently it returns  an in-person upcoming appointment with "At ${facilityName}" or "At VA facility". However, based on the pending AC, pending requests for an in-person should return "In Person" not "At VA facility".  Maybe create an overload for selectModalityText(appointment, isRequest) such that if isRequest and isInPerson then return "In Person"

**Review by jenniemc — DISMISSED** [review]
Just have a few changes for pending appointment list based on the AC. 

### Pending AC
- [x] 1st column: {type of care} (bold text)
- [x] 2nd column: {preferred modality}
- [x] 3rd column: {selected facility} for VA requests 
- [x] 3rd column {preferred provider name} for CC requests

### Modality text 
- [x] In Person
- [x] Video
- [x] Phone 
- [x] Community care

**Screenshoot based on local environment**

Screenshot 2023-03-13 at 6 10 43 PM

File: src/applications/vaos/appointment-list/components/AppointmentsPageV2/RequestAppointmentLayout.jsx (hunk 1 above)
Comment (vbahinwillit) [reply src/applications/vaos/appointment-list/components/AppointmentsPageV2/RequestAppointmentLayout.jsx:71]: Done.

**Review by jenniemc — APPROVED** [review]
LGTM!

**Pull request merged**